cli/attendance_weeklySummaryEmail.php

```diff
@@ -31,9 +32,9 @@
 } else {
     setCurrentSchoolYear($guid, $connection2);
 
-    require_once $_SESSION[$guid]['absolutePath'].'/modules/Attendance/moduleFunctions.php';
-    require_once $_SESSION[$guid]['absolutePath'].'/modules/Attendance/src/attendanceView.php';
-    $attendance = new Module\Attendance\attendanceView($gibbon, $pdo);
+    require_once __DIR__ . '/modules/Attendance/moduleFunctions.php';
```

Comment:
__DIR__ is in the cli directory in this instance
```suggestion
    require_once __DIR__ . '/../modules/Attendance/moduleFunctions.php';
```

Reply:
"moduleFunctions.php" is obsoleted here?